src/components/StaticMap.tsx

```diff
@@ -23,7 +36,7 @@ const StaticMap = (props: Coordinates) => {
           `${latitude}` +
           "," +
           `${longitude}` +
-          "&key=AIzaSyDZNQlSlEIkFAct5VzUtsP4dSbvOr2bE18"
```

Comment:
🙏  - but unfortunately this will remain in the git history forever. I think we plan to rotate at some point though so it won't really matter.

Reply:
indeed.